package.json

```diff
@@ -89,8 +91,10 @@
     "lru-cache": "^7.14.0",
     "methods": "^1.1.2",
     "module-details-from-path": "^1.0.3",
+    "msgpack-lite": "^0.1.26",
     "node-abort-controller": "^3.0.1",
     "opentracing": ">=0.12.1",
+    "pako": "^2.1.0",
```

Comment:
Why is this needed?

Reply:
This isn't needed anymore, it's leftover from some refactor